pkgs/development/python-modules/pytest-factoryboy/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, pythonPackages }:
```

This is in the python-modules dir, where most python dependencies should be explicitly declared as arguments.
So with formatting and this change, it should look something like...
```suggestion
{ stdenv
, fetchFromGitHub
, buildPythonPackage
, pytest
, inflection
, factory_boy
...
}:
```

---

Done.